pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/render/LanguageSettings.kt

```diff
@@ -42,6 +43,14 @@ val pensjonLatexSettings = languageSettings {
         )
     }
 
+    setting(LanguageSetting.Sakspart.vedleggGjelderNavn) {
+        Literal.create(
+            Bokmal to "Vedlegget gjelder:",
+            Nynorsk to "Vedlegget gjeld:",
+            English to "Attachment regarding:",
```

Comment:
Bruker vi ikke et annet ord på engelsk for vedlegg?

Reply:
Vi har brukt "Attachments:" for vedlegg listen så langt.

Reply:
Attachment er foretrukket over appendix. Krever litt mere tekst-vurderinger, så det kan tas i en annen branch.